app/constants/scheme-names.js

```diff
@@ -17,5 +20,8 @@ module.exports = {
   [CS]: 'Countryside Stewardship',
   [BPS]: 'BPS',
   [FDMR]: 'FDMR',
-  [MANUAL]: 'Manual Invoice'
+  [MANUAL]: 'Manual Invoice',
+  [ES]: 'Environmental Stewardship',
+  [FC]: 'Forestry Commission',
+  [IMPS]: 'IMPS'
```

Comment:
I noticed in one of the other repos these were just CS, Manual, ES etc. Which is the preferred option?

Reply:
Depends on what the values are used for.  The other one is the data hub which populates data in the dashboards where it was agreed to use the acronyms.

Here we're sending emails to the business and need to be a bit clearer about what the scheme is so we have the full name.

Reply:
That is a fair comment.

BPS = Basic Payment Scheme
FDMR = Financial Discipline Mechanism Reimbursement

However, the business tend to refer to those commonly as BPS and FDMR, whereas they refer to CS as Countryside Stewardship.

So agree it's not consistent, but it makes sense from a business perspective.

Reply:
I'll only cry a little - that's alright if it's better for the business :)